homework-submission/week2/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './components/Header';
+import TODOList from './components/TODOList';
+import Data from './TODOS.json'
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      items: Data,
+    }
+  }
+
+  checkStatus = (id) => {
+    const newItems = this.state.items.map((element) => {
+      if (id === element.id) {
+        return {
+          ...element,
+          done: !element.done
+        }
+      }
+      return element;
+    });
+    this.setState({ items: newItems });
+  }
+
+  render() {
+    const title = 'My TODO List';
+    return (
+      <div className="root">
+        <div className="App">
+          <Header />
+          <section>
+            <h1>{title}</h1>
+            {this.state.items.map((element, index) => {
+              return < TODOList
+                key={index}
+                TODO_Id={element.id}
```

Comment:
Follow the official style guide of Reactjs app. https://github.com/airbnb/javascript/tree/master/react. They use *camelCaseNaming* not underscore.